### PR TITLE
Add Stimulus

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

Draft to test https://github.com/sonata-project/form-extensions/pull/335

I am targeting this branch, because this is BC and will allow to be compatible with form-extensions 2.0.

## Changelog

```markdown
### Added
- Stimulus dependency and initialization inside main `app.js`
```

### CODE REVIEW

**Comment by eerison** [comment]
Hey @jordisala1991 just for my information

what is missing for this PR pass?

File: .github/workflows/frontend.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
             - name: Install Node
               uses: actions/setup-node@v3
               with:
-                  node-version: '16'
```
Comment (VincentLanglet) [inline .github/workflows/frontend.yaml]: Need to be updated on dev-kit no ?

File: assets/scss/styles.scss [hunk 2]
```diff
@@ -13,6 +13,7 @@
  */
 
 html {
+  font-size: initial !important;
```
Comment (jordisala1991) [inline assets/scss/styles.scss]: this is only needed if we end up using flatpickr. Otherwise it might now be needed. Maybe it is better to revert that for now.

File: .stylelintrc.js [hunk 3]
```diff
@@ -15,7 +15,7 @@
 
 module.exports = {
   customSyntax: 'postcss-scss',
-  extends: ['stylelint-config-standard', 'stylelint-config-prettier'],
```
Comment (VincentLanglet) [inline .stylelintrc.js]: Should be on devkit (and what's the reason behind this change)

File: assets/scss/styles.scss [hunk 4]
```diff
@@ -13,6 +13,7 @@
  */
```
Comment (VincentLanglet) [inline assets/scss/styles.scss]: What's the reason behind all the styles changes ?

**Comment by jordisala1991** [comment]
@VincentLanglet wdyt about this one? IMO, for now one of the best alternatives in Symfony to deal with large amounts of js, specially if the dom can be dinamically created (like our modals).

So, it will be useful not only to the form-extensions but we should be able to rewrite everything to use Stimulus controllers.

File: .github/workflows/frontend.yaml (hunk 1 above)
Comment (jordisala1991) [reply .github/workflows/frontend.yaml]: Yes, I will provide a dev-kit Pr too.

File: .stylelintrc.js (hunk 3 above)
Comment (jordisala1991) [reply .stylelintrc.js]: the plugin for prettier is deprecated since last stylelint version (all rules related got deprecated in core, so it does not conflict anymore)

File: assets/scss/styles.scss (hunk 4 above)
Comment (jordisala1991) [reply assets/scss/styles.scss]: Newer versions of stylelint standard config, have new rules, like when you upgrade php cs fixer and some code needs fixing.

File: assets/scss/styles.scss (hunk 2 above)
Comment (jordisala1991) [reply assets/scss/styles.scss]: Removed for now, until we add flatpickr, if needed.

**Review by VincentLanglet — APPROVED** [review]
